modules/filtering/CMakeLists.txt

```diff
@@ -28,7 +28,8 @@ if(NOT EMSCRIPTEN)
             # Hold off on using MKL for now to avoid bloating the package size.
             # include(blas) # Accelerate on arm64 macOS, MKL on other platforms
             # target_compile_definitions(lagrange_filtering PRIVATE LA_SOLVER_MKL)
-            add_library(BLAS::BLAS GLOBAL IMPORTED INTERFACE)
+            add_library(BLAS IMPORTED INTERFACE GLOBAL)
```

Comment:
Wait why is that an error before? Imported targets can use the `::` syntax afaik.

Reply:
Could you try with `add_library(BLAS::BLAS INTERFACE IMPORTED GLOBAL)` instead of `GLOBAL IMPORTED INTERFACE`?

Reply:
I suspect `GLOBAL` got interpreted as a source file instead of keyword.